packages/yarnpkg-fslib/sources/LazyFS.ts

```diff
@@ -22,6 +22,10 @@ export class LazyFS<P extends Path> extends ProxiedFS<P, P> {
     return this.instance;
   }
 
+  protected set baseFs(value) {
+    this.instance = value;
```

Comment:
I don't see that being used in the PR, is it still needed?

Reply:
Yep, it is needed for the babel, otherwise it complains that it cannot set baseFs in derived classes